Resources/config/services.xml

```diff
@@ -7,10 +7,19 @@
     <parameters>
         <parameter key="elao.form_translation.tree_builder.class">Elao\Bundle\FormTranslationBundle\Builders\FormTreeBuilder</parameter>
         <parameter key="elao.form_translation.key_builder.class">Elao\Bundle\FormTranslationBundle\Builders\FormKeyBuilder</parameter>
+        <parameter key="elao.form_translation.extractor.form_extractor.class">Elao\Bundle\FormTranslationBundle\Translation\Extractor\FormExtractor</parameter>
     </parameters>
 
     <services>
         <service id="elao.form_translation.tree_builder" class="%elao.form_translation.tree_builder.class%" />
         <service id="elao.form_translation.key_builder" class="%elao.form_translation.key_builder.class%" />
+        <service id="elao.form_translation.extractor.form_extractor" class="%elao.form_translation.extractor.form_extractor.class%" public="false">
```

Comment:
We should move all JMS related service and parameter declaration in a `jms_translation.xml`.
This way we can import this config file, and declare associated services, only when needed.